### PR TITLE
Added some questions to security-quiz.ts related to cloud computing secutity

### LINKED ISSUE #1135: Adding Security questions to quiz site
### What general issue would you like to create?

What general issue would you like to create?

[developerquiz.org](https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/developerquiz.org) currently has over 1500 quiz questions. We are looking to expand on the security questions and we encourage other developers to add their quiz questions to the site.

You can find the complete list of security questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/security-quiz.ts

You can add your own questions to the top of that file.
Please first check to make sure your question doesn't already exist in the file before creating a PR.

Here is an example format for the questions.

```
 {
    Question: "Which of the following is not an Attack model in cryptanalysis?",
    Answer: "CVA (Chosen Vector Attack)",
    Distractor1: "CPA (Chosen Plaintext Attack)",
    Distractor2: "CCA (Chosen Ciphertext Attack)",
    Distractor3: "COA (Ciphertext Only Attack)",
    Explanation: "There is no attack model named CVA (Chosen vector attack).",
    Link: "https://en.wikipedia.org/wiki/Attack_model"
  }
```
For the link field, please make sure to use a freeCodeCamp article, freeCodeCamp YouTube video or official documentation.
If you choose to reference a video, please make sure to include the timestamp for the topic covered.

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#).

There are plenty of freeCodeCamp news articles to pull Security questions from:
https://www.freecodecamp.org/news/tag/security

This issue will not be assigned to anyone and will remain open for multiple contributors.
Please do not assign yourself to this issue or close it.

Happy contributing!

### PR DESCRIPTION
**Pull request by tsr94**
## Summary of changes

Added a set of security-related multiple-choice questions to the security-quiz.ts file. The questions are from cloud cyber security.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

related to #1135

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
Please do not close the issue, as per written in the issue. Instead write "Related to"

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -1,11 +1,53 @@
 const securityQuiz = [
   {
-    Question:"What is the primary purpose of a wordlist in ethical hacking ?",
-    Answer : "To automate password cracking attempts",
+    Question:
+      "Which of the following is an example of a denial-of-service (DoS) attack??",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:4]:
```suggestion
      "Which of the following is an example of a denial-of-service (DoS) attack?",
```

File: src/data/security-quiz.ts [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
+    Link: "https://en.wikipedia.org/wiki/Cloud_computing_security"
+  },
+  {
+    Question: "Which standard is widely adopted for cloud cybersecurity?",
+    Answer: "ISO/IEC 27001",
+    Distractor1: "GDPR",
+    Distractor2: "ITIL",
+    Distractor3: "PCI DSS",
+    Explanation:
+      "ISO/IEC 27001 is an internationally recognized standard for information security, including cloud environments",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:20]:
```suggestion
      "ISO/IEC 27001 is an internationally recognized standard for information security, including cloud environments.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
A few grammar and punctuation fixes but otherwise looks good

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by tsr94** [comment]
Thank you [jdwilkin4](https://github.com/jdwilkin4), Very happy

**Pull request merged**